.github/workflows/publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Push on main
```

Comment:
This isn't a push, it's a publish using a button

Reply:
changed